test/slack.test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const assert = require('chai').assert;
+const mockery = require('mockery');
+const sinon = require('sinon');
+
+sinon.assert.expose(assert, { prefix: '' });
+describe('slack', () => {
+    let configMock;
+    let channels;
+    let payload;
+    let slacker;
+    let WebClientMock;
+    let WebClientConstructorMock;
+
+    before(() => {
+        mockery.enable({
+            useCleanCache: true,
+            warnOnUnregistered: false
+        });
+    });
+
+    beforeEach(() => {
+        WebClientMock = {
+            channels: {
+                list: sinon.stub().resolves({
+                    channels: [{
+                        name: 'meeseeks',
+                        id: '23'
+                    }, {
+                        name: 'dd',
+                        id: '21'
+                    }]
+                })
+            },
+            chat: {
+                postMessage: sinon.stub().resolves({})
+            }
+        };
+        WebClientConstructorMock = sinon.stub().returns(WebClientMock);
+        mockery.registerMock('@slack/client', WebClientConstructorMock);
+
+        // eslint-disable-next-line global-require
+        slacker = require('../slack');
+    });
+
+    afterEach(() => {
+        mockery.deregisterAll();
+        mockery.resetCache();
+    });
+
+    after(() => {
+        mockery.disable();
+    });
+
+    describe('notifier listens to server emits', () => {
```

Comment:
This description is not right, can you change it to `slacker posts message to channels`?